### DISCUSSION TITLE
NavigationViewItem has a different default color on Desktop/Skia than WinAppSDK

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#16654] [Q&A] [answered]
### Current behavior

The menu item texts are invisible

![image](https://github.com/unoplatform/uno/assets/1608424/77c1af4e-9115-4442-9af3-fd0358b45c39)

### Expected behavior

The menu item texts should be visible

### How to reproduce it (as minimally and precisely as possible)

Run this project [Test.zip](https://github.com/unoplatform/uno/files/15256989/Test.zip)

### Workaround

Add Foreground="Black" to each menu item

### Works on UWP/WinUI

Yes

### NuGet package version(s)

5.2.108

### Affected platforms

Skia (WPF)

**Comment by agneszitte** [comment]
fyi @MartinZikmund, @Youssef1313

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report! The requested theme is not yet supported on elements in uno (https://github.com/unoplatform/uno/issues/3302), which explains the difference in behavior you're seeing as Uno applies the app-level theme on the `NavigationView`.

If you want to force the light theme, set the `RequestedTheme` property in `App.xaml`.

**Reply by ArchieCoder** [reply]
I confirm that forcing the theme in App fulfills my request. Thanks.
